crates/kit/src/to_disk.rs

```diff
@@ -247,15 +247,63 @@ impl ToDiskOpts {
                 tty=--tty
             fi
 
+            # Workaround for issue #126: Override container policy to allow signature changes.
+            # Some images (e.g., RHEL) have strict signature policies that prevent bootc install
+            # from changing layer representation. We override /etc/containers/policy.json with
+            # a permissive policy that allows all operations.
+            export STORAGE_OPTS=additionalimagestore=${AIS}
+            SOURCE_REF={SOURCE_IMGREF}
+            
+            # Create permissive policy.json (use /var/tmp since it's mounted into podman container)
+            # Mount as directory to /etc/containers so podman creates the directory if it doesn't exist
+            POLICY_DIR=/var/tmp/bcvk-policy-dir
```

Comment:
I know this is an ephemeral isolated system but it's probably still good to avoid hardcoded filenames in /tmp or /var/tmp. So we could use e.g. `policydir=$(mktemp -d)` or so.

It might be a nicer cleanup to have this policy JSON string as a literal file in the Rust source that we include via `include_str!` and then template into the shell script.